rpc/commands/commands.proto

```diff
@@ -230,3 +233,19 @@ message VersionResp {
   // The version of Arduino CLI in use.
   string version = 1;
 }
+
+message SketchLoadReq {
+  // Absolute path to single sketch file or a sketch folder
+  string sketch_path = 1;
+}
```

Comment:
This should also include the `instance`.
```
message SketchLoadReq {
  Instance instance = 1;
  // Absolute path to single sketch file or a sketch folder
  string sketch_path = 2;
}
```

Reply:
Sure!

Reply:
Well, I do not know if it's required, but strange to not having symmetric APIs. @cmaglie, can you please help with this? Thank you!

Reply:
I didn't include it because it's actually not used by that command but I guess it costs nothing adding it.

Reply:
Works well so far, great simplification for the IDE. Thanks!